meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-boot.mount

```diff
@@ -7,7 +7,7 @@ Conflicts=umount.target
 [Mount]
 What=/dev/disk/by-label/resin-boot
 Where=/mnt/boot
-Type=vfat
+Type=ubifs
```

Comment:
@michal-mazurek @floion @agherzan 

Just Sent a New [PR](https://github.com/resin-os/meta-resin/pull/756) 

UBIFS Class has been moved into image_types_resin.bbclass.

The UDev rules have been Removed because they are being Append [Here](https://github.com/resin-os/resin-chip/blob/master/layers/meta-resin-chip/recipes-core/packagegroups/packagegroup-resin.bbappend)

The *.mount files are unmodified/Set to Auto